frontend/src/scenes/session-recordings/player/modal/SessionPlayerModal.tsx

```diff
@@ -15,33 +15,21 @@ import { sessionPlayerModalLogic } from './sessionPlayerModalLogic'
  *
  */
 export function SessionPlayerModal(): JSX.Element | null {
-    const { activeSessionRecording } = useValues(sessionPlayerModalLogic())
+    const { activeSessionRecordingId, matchingEventsMatchType } = useValues(sessionPlayerModalLogic())
     const { closeSessionPlayer } = useActions(sessionPlayerModalLogic())
 
-    // activeSessionRecording?.matching_events should always be a single element array
-    // but, we're filtering and using flatMap just in case
-    const eventUUIDs =
```

Comment:
i'm surprised this is safe... but i don't have enough recordings locally to test

there are two matching modes iirc

1. simple events -> done locally
2. events with filters -> done as a follow-up query

could this path be necessary for that second type?